Source/WebKit/WebProcess/WebPage/WebCookieJar.cpp

```diff
@@ -98,7 +98,7 @@ static bool shouldBlockCookies(WebFrame* frame, const URL& firstPartyForCookies,
         return false;
     }
 
-    return true;
+    return !shouldAttemptHandlingPartitionedCookies;
```

Comment:
Why would this not return true? If client specifies to block all third-party cookies (i.e. `WebProcess::singleton().thirdPartyCookieBlockingMode() == ThirdPartyCookieBlockingMode::All` above), then we don't need to care whether the cookie would be put in partitioned storage or non-partitioned storage, right?

For the initial implementation, I think we don't need to change the check in web process side.

Reply:
I am interpreting `ThirdPartyCookieBlockingMode::All` as "unpartitioned first-party cookies in a third-party context", instead of "all cookies available in a third-party context".  But we can start with continuing to block all cookies in a third-party context, as you suggested.